cmd/nodecmd/create.go

```diff
@@ -321,6 +321,19 @@ func createNode(_ *cobra.Command, args []string) error {
 	// Create new EC2 client
 	instanceID, elasticIP, certFilePath, keyPairName, err := createEC2Instance(rootBody, ec2Svc, hclFile, region, ami, certName, prefix, securityGroupName)
 	if err != nil {
+		ux.Logger.PrintToUser("Failed to create AWS cloud server, please try creating again in a different region")
```

Comment:
maybe you suggest to use a different region, but maybe the solution is other one? how can we sure
the solution is to use a different region?

Reply:
ie: can we check that the error is EIP limit? can createEC2Instance notify that?